app/src/main/res/values/styles_textview.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It could be better if we follow Material type system.

Reply:
Can you give an example of what needs to be changed @necatisozer ?

Reply:
Rather than using type specs(size, weight, color) in style names, we need an explanatory naming system here.

Example: https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/type.xml

Reply:
It could be better than this naming in the small app. Because, We would have a few different values. but, we can have many different values in the growing app. So I think, current state is acceptable, also this naming is readable and clear in the xml file.

Reply:
I will know easily the TextView has textSize is 14 and textStyle is bold when I set "TextView.Bold.14"

Reply:
Then, styling becomes meaningless. If we want to support Dark Mode for example, we will have some serious problems.
Anyway, for rapid development, this can be merged like this. We can refactor it in a proper time.

Reply:
@oshamahue What do you thing?

Reply:
I agree with both of you. We don't have a standard defined prior to this conversation. The material approach seems to be better defined. But the scope of the project is small. I leave it up to @perihanimamoglu to decide.